src/orderbook/index.ts

```diff
@@ -239,20 +249,24 @@ async function batchExchangeDeployment(web3: Web3): Promise<[BatchExchange, Tran
   return [contract, tx]
 }
 
-(async () => {
-  try {
-    const { ETHEREUM_NODE_URL, ORDERBOOK_END_BLOCK } = process.env
-    if (!ETHEREUM_NODE_URL) {
-      throw new Error('missing required evironment variable ETHEREUM_NODE_URL')
-    }
+if (typeof jest === 'undefined') {
```

Comment:
Maybe it makes sense to pull the streamed orderbook class into its own file and then have the index.ts not worry about jest?